### PR TITLE
solution date reformating

### PR DESCRIPTION
**Pull request by Evgenygroup**
#8

### CODE REVIEW

**Comment by Evgenygroup** [comment]
solved via stream

File: DateReformat/DateReformatString.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+
+    public static List<String> reformatDates(List<String> input) {
+
+        List<String> reformatedDate = input
+                .stream()
+                .map(LocalDate::parse)
+                .map(s -> s.format(DateTimeFormatter.ofPattern
+                        ("dd MMMM,yyyy EEEE", Locale.ENGLISH)))
```
Comment (sergeylukichev) [inline DateReformat/DateReformatString.java:28]: You may define a variable of type DateTimeFormatter before and use it:
`DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd MMMM,yyyy EEEE", Local.ENGLISH)l`

**Comment by sergeylukichev** [comment]
👍

File: DateReformat/DateReformatString.java (hunk 1 above)
Comment (Evgenygroup) [reply DateReformat/DateReformatString.java:28]: ok, thanks

**Pull request merged**